### ISSUE TITLE
SIGSEGV when trying to start iostat.

### ISSUE THREAD

**Issue by lesovsky** [lesovsky/pgcenter#28]
pgCenter crashes when trying to start iostat (B hotkey). 
gdb output

```
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x0000000000407bfe in print_iostat (window=0x11b3380, w_cmd=0x11ac260, c_ios=0x7ffcb2b95100, p_ios=0x7ffcb2b94ff0, bdev=32, repaint=0x7ffcb2b9528f) at pgcenter.c:1689
1689            c_ios[i]->major = major;
```

Catched on systems with too many lines in /proc/diskstats.

**Comment by lesovsky** [comment]
Ghhm, strange but not confirmed completely.
1. pgcenter crashes when using fake diskstats (diskstats copied using copy-paste), but ok if copied with scp.
2. also it work correctly with really huge diskstats.

If problem occurs again, check the diskstats file format.

**Comment by lesovsky** [comment]
Found again and resolved with cad30df
